### PR TITLE
➖ unused components

### PR DESCRIPTION
**Pull request by roiLeo**
- part of #4750  
- remove unused components

### CODE REVIEW

File: components/shared/SimpleAccountBalance.vue
Comment (vikiival) [inline components/shared/SimpleAccountBalance.vue:1]: How is this sorted now?

File: components/bsx/input/OfferBalanceInput.vue
Comment (vikiival) [inline components/bsx/input/OfferBalanceInput.vue:1]: How is this sorted now?

File: components/bsx/Offer/CollectionOffers.vue
Comment (vikiival) [inline components/bsx/Offer/CollectionOffers.vue:1]: Did we ditched this screen or how do we handle this?

**Review by vikiival — COMMENTED** [review]
~~Few~~

File: components/bsx/Offer/CollectionOffers.vue
Comment (roiLeo) [reply components/bsx/Offer/CollectionOffers.vue:1]: Cant find related import

Comment (vikiival) [reply components/bsx/Offer/CollectionOffers.vue:1]: The ref I found:

- https://github.com/kodadot/nft-gallery/pull/3484

cc @exezbcz do we plan to show this?

File: components/bsx/input/OfferBalanceInput.vue
Comment (roiLeo) [reply components/bsx/input/OfferBalanceInput.vue:1]: I think we use `TokenBalanceInput` for now

File: components/bsx/Offer/CollectionOffers.vue
Comment (roiLeo) [reply components/bsx/Offer/CollectionOffers.vue:1]: delete-this-sponge-bob-square-pants

File: components/bsx/input/OfferBalanceInput.vue
Comment (vikiival) [reply components/bsx/input/OfferBalanceInput.vue:1]: Never used imo 😅 

- https://github.com/kodadot/nft-gallery/pull/3784

**Review by Jarsen136 — APPROVED** [review]

File: components/shared/SimpleAccountBalance.vue
Comment (roiLeo) [reply components/shared/SimpleAccountBalance.vue:1]: idk I think we use MultiAccountBalance instead

Comment (vikiival) [reply components/shared/SimpleAccountBalance.vue:1]: - https://github.com/kodadot/nft-gallery/commit/80164d986a9c24c835b5824f3475c1089419a0ed

**Review by vikiival — APPROVED** [review]

**Pull request merged**